src/test/java/seedu/exercise/logic/parser/ParserUtilTest.java

```diff
@@ -221,4 +222,101 @@ public void parseMuscles_collectionWithValidMuscles_returnsMuscleSet() throws Ex
 
         assertEquals(expectedMuscleSet, actualMuscleSet);
     }
+
+    @Test
+    public void parseFullName_null_throwsNullPointerException() {
```

Comment:
naming can be improved. something like `parseFullName_withNull_throwsNullPointerException`

Reply:
Renamed the test methods